inc/answer.class.php

```diff
@@ -65,4 +65,26 @@ public static function canView() {
    public static function getTypeName($nb = 0) {
       return _n('Answer', 'Answers', $nb, 'formcreator');
    }
+
+   /**
+    * Define how to display a specific value in search result table
+    *
+    * @param  string $field   Name of the field as define in $this->getSearchOptions()
+    * @param  mixed  $values  The value as it is stored in DB
+    * @param  array  $options Options (optional)
+    * @return mixed           Value to be displayed
+    */
+   public static function getSpecificValueToDisplay($field, $values, array $options = []) {
```

Comment:
@orthagh : I requested your review because of this method. I don't have a better way to properly format some field types. This implementation may have low performance on big lists, but the format depends on metadata of the field type.

How it works ? the method does not has any argument providing a question ID. Then to determine the field type I use a search option providing the ID of the answer instead of the answer itself.

The row in the DB contains the question ID and give access to the actual field type. Then I use the methods dedicated to output the answer in a readable format.

Reply:
example:
https://github.com/glpi-project/glpi/blob/9.4/bugfixes/inc/commonitilobject.class.php#L2670-L2693

this method is always called for the current object (and the corresponding table), so you just need to filter on the field

Reply:
The method belongs to PluginFormcreatorAnswer, and the context is static then I cannot use an instance to find the PluginFormcreatorQuestion object containing the type of field. 
I have several cases 
- a string to be displayed as is
- a number which is an ID of an other itemtype (glpi object, dropdown fields)
- a number to convert into a string from an array (urgency field)
- a set of values separated by \r\n (checkboxes, multiselecti fields)

Only the 2 first can be processed without this method. The 2 last cases needs a different processing which require that I can determine the type of field stored in the associated PluginFormcreatorQuestion instance.

By the way, maybe should I protect the trnasformation by testing the column name.